shop3/docs/manual.texinfo

```diff
@@ -2889,28 +3024,28 @@ the list of protected conditions.
 
 Here is an example:
 
-@multitable {Result(@emph{S,h@sup{u}})result(@emph{S,o@sup{u}})} {(:operator (!set-money ?person ?old ?new)((has-money ?person ?old))((has-money ?person ?old))((has-money ?person ?new)))} 
-@item 
+@multitable {Result(@emph{S,h@sup{u}})result(@emph{S,o@sup{u}})} {(:operator (!set-money ?person ?old ?new)((has-money ?person ?old))((has-money ?person ?old))((has-money ?person ?new)))}
```

Comment:
Is the Result in the multitable more properly a `@var` ?
And similarly at 3029, 3035, 3045, 3106-3109